src/features/MoviePage/Content/MainInfo/styled.js

```diff
@@ -2,47 +2,103 @@ import styled from "styled-components";
 import { ReactComponent as StarImage } from "../../../../images/Vector.svg";
 
 export const Container = styled.div`
-  min-height: 544px;
+  min-height: 350px;
   display: flex;
   align-items: center;
   gap: 40px;
   background-color: ${({ theme }) => theme.color.white};
   box-shadow: 0px 4px 12px 0px #bac7d580;
-  padding-left: 40px;
+  padding: 40px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.tablet}) {
+    align-items: flex-start;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.largeMobile}) {
+    padding: 16px;
+    gap: 16px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.smallMobile}) {
+    max-width: 300px;
+    min-height: 350px;
+  }
 `;
 
 export const Post = styled.img`
   width: 312px;
   height: 464px;
   border-radius: 5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.tablet}) {
+    width: 200px;
+    height: 275px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mediumMobile}) {
+    width: 114px;
+    height: 169px;
+  }
 `;
 
 export const Description = styled.div`
   display: flex;
   flex-direction: column;
-  margin: 48px 40px 92px 0;
+  margin-bottom: 20px;
   gap: 24px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.tablet}) {
+    -size: 30px;
```

Comment:
I think you missed font-size here in Description form tablet media query

Reply:
Yeah, you're right! Thanks, that you note this) Had been corrected)